style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  display: flex;
+  justify-content: center;
+}
+
+body {
+  color: #091e42;
+  background-color: #f7f7f9;
+  font-family: 'Poppins', sans-serif;
+  width: 375px;
+}
+
+header {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  width: 375px;
+  height: 48px;
+  box-sizing: border-box;
+}
+
+#hamburger-btn {
+  box-sizing: border-box;
+  position: absolute;
+  background-image: url("Union.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 8px;
+  border-color: transparent;
+  background-color: transparent;
+  padding: 19px 19px 0 0;
+  right: 24px;
+  width: 40px;
+  height: 40px;
+}
+
+#logo {
+  left: 24px;
+  padding: 14px 0;
+  display: flex;
+  align-items: center;
+  position: absolute;
+  box-sizing: border-box;
+}
+
+h1 {
+  position: static;
+  width: 327px;
+  height: 150px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+}
+
+p {
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+
+  /* Inside auto layout */
+
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0;
+}
+
```

Comment:
[Optional]
Kindly use the CSS classes when you want to style HTML elements because CSS classes are reusable. If you  style the HTML tag and you want to use `<h1>` or `<p>` tag for something else with a different style it'll make issues for you. 👍🏻